src/Microsoft.AspNetCore.Server.Kestrel.Core/Internal/FrameConnection.cs

```diff
@@ -123,7 +124,7 @@ public void Timeout()
                     _frame.Input = _adaptedPipeline.Input.Reader;
                     _frame.Output = _adaptedPipeline.Output;
 
-                    _adaptedPipelineTask = RunAdaptedPipeline();
+                    var ignore = RunAdaptedPipeline();
```

Comment:
Is it possible for this code path to throw and `_adaptedPipelineTcs` never get set?